app_legacy/Helpers/render/game.php

```diff
@@ -385,6 +385,18 @@ function RenderRecentGamePlayers(array $recentPlayerData): void
             $userName,
             $activity
         );
+
+        // Check if $activity contains a message about an "Unknown macro", and
+        // if so, strip the RP and replace it with an outdated emulator warning.
+        if (mb_strpos($activity, 'Unknown macro') !== false) {
+            $activity = <<<HTML
+                <div class="flex items-center gap-x-1" title="Outdated emulator detected!">
+                    <span>⚠️</span>
+                    <span>Playing $gameTitle</span>
+                </div>
+            HTML;
```

Comment:
I am not sure if this should also say "Playing $gameTitle with an outdated emulator" just like the homepage active players widget, or if just using a tooltip around the whole div is good enough.

Here's what it looks like with the additional copy, kind of gross with the emoji floating top left imo, but I could be being too nitpicky:

![Screenshot 2023-04-16 at 3 26 33 PM](https://user-images.githubusercontent.com/3984985/232337368-9379b63d-fad1-4dce-b59a-69beec4d61f8.png)

Reply:
You don't even need "with an outdated emulator" to make it wrap, though mine seems to be wrapping separately from the icon.
![image](https://user-images.githubusercontent.com/32680403/233167697-bb871396-0708-46fb-a12e-bd76755ebdb9.png)

Reply:
I've addressed the wrapping issue in https://github.com/RetroAchievements/RAWeb/pull/1482/commits/408191873ff019f06cab20525699f1facdf54c3b.